src/lib/utils.ts

```diff
@@ -405,7 +405,10 @@ export function getTransition(entity: Zh.Endpoint | Zh.Group, key: string, meta:
         const time = toNumber(message.transition, 'transition');
         return {time: time * 10, specified: true};
     } else if (options.hasOwnProperty('transition')) {
```

Comment:
```suggestion
    } else if (options.hasOwnProperty('transition') && options.transition !== '') {
```

transition == 0 is something different than no transition (`''`), I propose to make this change instead